src/LudoGameApp/GameEngine/LudoEngine.cs

```diff
@@ -0,0 +1,280 @@
+﻿using System;
+using System.Collections.Generic;
+
+
+namespace GameEngine
+{
+    public class LudoEngine
+    {
+        public int Counter = 0;
+        private int LastDiceThrow { get; set; }
+        private int nrOfPlayer;
+        public bool OkToStart { get; set; }
+        public int NrOfPlayer
```

Comment:
Why is this a public property? Is it used or planed to be used in code implementing LudoEngine?